systemtests/tests/restore-test/testrunner-restore-file

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+set -e
+set -o pipefail
+set -u
+#
+# restore some single files from archives
+#
+TestName="$(basename "$(pwd)")"
+export TestName
+
+JobName=backup-bareos-fd
+
+#shellcheck source=../environment.in
+. ./environment
+
+#shellcheck source=../scripts/functions
+. "${rscripts}"/functions
+
+start_test
+
+file1="$(pwd)/tmp/data/weird-files/socket.pl"
+file2="$(pwd)/tmp/data/weird-files/softlinkee"
+file3="$(pwd)/tmp/data/weird-files/softlink-to-softlinkee"
+for f in "$file1" "$file2" "$file3" ; do
+    if [[ -f "$tmp/bareos-restores/$f" ]]; then
+	rm "$tmp/bareos-restores/$f"
+    fi;
+done
+cat <<END_OF_DATA >"$tmp/bconcmds"
+@$out /dev/null
+setdebug level=100 storage=File
+@$out /tmp/log1.out
+status director
+status client
+status storage=File
+wait
+messages
+@$out $tmp/jobs.out
+list jobs
+@$out $tmp/should-work.out
+restore client=bareos-fd fileset=SelfTest where=$tmp/bareos-restores \
+	file=$file1 \
+	done yes
+messages
+wait
+@$out $tmp/restore-from-selected-job.out
+restore
+12
+1
+mark $file2
+done
+yes
+wait
+messages
+@$out $tmp/restore-from-list-and-time.out
+restore
+8
+$(( $(date +%Y) + 1))
+2025
```

Comment:
2025 can be removed ?

Reply:
That was an error.  Was fixed!

Reply:
See new review, the error was present two times ;-)